arcor2_AREditor/Assets/TABLET/Scripts/GUI/OrientationJointsDetailMenu.cs

```diff
@@ -54,7 +54,7 @@ public class OrientationJointsDetailMenu : MonoBehaviour, IMenu {
         if (joints != null && joints.Id == args.Data.Id) {
             joints = args.Data;
             UpdateMenu();
-            MoveHereModel();
+            //MoveHereModel();

```

Comment:
Remove it or add explaining text on why it is commented out.

Reply:
Removed it